### PR TITLE
perf: conver tick and exponent at price one to int64

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

Convert pool's tick and exponent at price one to int64 for performance and readability

### CODE REVIEW

File: x/concentrated-liquidity/tick.go [hunk 1]
```diff
@@ -327,13 +324,13 @@ func (k Keeper) GetTickLiquidityNetInDirection(ctx sdk.Context, poolId uint64, t
...
 
-	ctx.Logger().Debug(fmt.Sprintf("userGivenStartTick %s, boundTick %s, currentTick %s\n", userGivenStartTick, boundTick, p.GetCurrentTick()))
+	ctx.Logger().Debug(fmt.Sprintf("userGivenStartTick %s, boundTick %s, currentTick %d\n", userGivenStartTick, boundTick, p.GetCurrentTick()))
 
 	startTick := p.GetCurrentTick()
 	// If start tick is set, use it as the current tick for grabbing liquidities from.
 	if !userGivenStartTick.IsNil() {
-		startTick = userGivenStartTick
-		ctx.Logger().Debug(fmt.Sprintf("startTick %s set to user given\n", startTick))
+		startTick = userGivenStartTick.Int64()
```
Comment (czarcas7ic) [inline x/concentrated-liquidity/tick.go:332]: Is there a reason we don't switch userGivenStartTick to an int64 as well?

File: x/concentrated-liquidity/tick.go [hunk 2]
```diff
@@ -368,12 +365,12 @@ func (k Keeper) GetTickLiquidityNetInDirection(ctx sdk.Context, poolId uint64, t
...
 	}
 
-	ctx.Logger().Debug(fmt.Sprintf("currentTick %s; current tick's sqrt price%s\n", currentTick, currentTickSqrtPrice))
+	ctx.Logger().Debug(fmt.Sprintf("currentTick %d; current tick's sqrt price%s\n", currentTick, currentTickSqrtPrice))
 
 	// function to validate that start tick and bound tick are
 	// between current tick and the min/max tick depending on the swap direction.
 	validateTickIsInValidRange := func(validateTick sdk.Int) error {
-		_, validateSqrtPrice, err := math.TickToSqrtPrice(validateTick)
+		_, validateSqrtPrice, err := math.TickToSqrtPrice(validateTick.Int64())
```
Comment (czarcas7ic) [inline x/concentrated-liquidity/tick.go:373]: Same here with validateTick

**Review by czarcas7ic — APPROVED** [review]
Looks good!

File: x/concentrated-liquidity/tick.go (hunk 1 above)
Comment (p0mvn) [reply x/concentrated-liquidity/tick.go:333]: Will be done later. I will investigate and refactor this query after swaps

**Pull request merged**